### PR TITLE
Post checks trajectories in trajopt_planner for collisions

### PR DESCRIPTION
**Pull request by mpowelson**

### CODE REVIEW

**Comment by mpowelson** [comment]
Also, trajopt_planner does have rosconsole in it.

File: tesseract/tesseract_planning/tesseract_planners/src/trajopt/trajopt_planner.cpp [hunk 1]
```diff
@@ -122,7 +130,7 @@ bool TrajOptPlanner::solve(PlannerResponse& response, const TrajOptPlannerConfig
     response.status_description = status_code_map_[0] + ": " + sco::statusToString(opt.results().status);
   }
 
-  return true;
+  return (response.status_code > 0);
```
Comment (Levi-Armstrong) [inline tesseract/tesseract_planning/tesseract_planners/src/trajopt/trajopt_planner.cpp]: Should this be `>=`?

Comment (Levi-Armstrong) [reply tesseract/tesseract_planning/tesseract_planners/src/trajopt/trajopt_planner.cpp]: Friendly Reminder :)

Comment (mpowelson) [reply tesseract/tesseract_planning/tesseract_planners/src/trajopt/trajopt_planner.cpp]: Just fixed it.

**Review by Levi-Armstrong — APPROVED** [review]

**Pull request merged**